server/types.ts

```diff
@@ -1,14 +1,12 @@
 import { actionCreatorFactory } from 'typescript-fsa'
 
 import { Server, Action, LoguxSubscribeAction } from '..'
-import pino = require('pino')
 
 let server = new Server<{ locale: string }>(
   Server.loadOptions(process, {
     subprotocol: '1.0.0',
-    reporter: 'human',
     supports: '1.x',
-    logger: pino({ name: 'logux' }),
```

Comment:
Why do we have this change?

Reply:
Both `reporter: 'human'` and `logger` were specified here when I switched to pino, and than I didn't notice, that one of them is ignored. 
I thought that `'human'` logger should be left.
If I'm wrong I think we can use `'json'` to avoid "implementation details"

Reply:
We need to test that we defined `pino` types correctly for `logger` option.

Reply:
Hm, seems that I don't understand purpose of this file. 
Does it contain code just to check that no errors are emitted during type check?

Reply:
It is a test file for `.d.ts` type definitions. Check out https://github.com/ai/check-dts

Reply:
Thanks!